pkg/backup/s3/s3.go

```diff
@@ -46,18 +46,19 @@ func New(bucket, prefix string) (*S3, error) {
 	})
 }
 
+// MakePrefix concatenates prefix, "v1", namespace, clusterName to a single prefix path.
+func MakePrefix(prefix, namespace, clusterName string) string {
```

Comment:
This shouldn't be in this package.

If we want to make "backup/s3" agnostic of any k8s concept and only use prefix.